azure-pipelines.yml

```diff
@@ -74,8 +74,8 @@ jobs:
         $branchName = $env:BUILD_SOURCEBRANCH
 
         if ($branchName -eq "refs/heads/main") {
-            Write-Host "##vso[task.setvariable variable=TelemetryType]"{{ parameters.TelemetryType }}
-            Write-Host "Done setting the sign type and telemetry key instance for main branch"
+            Write-Host "##vso[task.setvariable variable=TelemetryType] ${{ parameters.TelemetryType }}"
```

Comment:
Is the space before the $ needed or will it cause issue?

Reply:
we can test and see via builds, waiting for them to run.

Reply:
Yes it is needed to get the value for the parameters. I refer other places they have spaces.

Reply:
@elenavillamil you were correct the space is not needed, corrected them will test and keep you posted, thanks for catching this up. :)